packages/@lwc/ssr-compiler/src/compile-template/transformers/element.ts

```diff
@@ -41,17 +42,31 @@ const bYield = (expr: EsExpression) => b.expressionStatement(b.yieldExpression(e
 // TODO [#4714]: scope token renders as a suffix for literals, but prefix for expressions
 const bConditionalLiveYield = esTemplateWithYield`
     {
-        const shouldRenderScopeToken = ${/* isClass */ is.literal} &&
+        const attrName = ${/* attribute name */ is.literal};
+        let attrValue = ${/* attribute value expression */ is.expression};
+        const isHtmlBooleanAttr = ${/* isHtmlBooleanAttr */ is.literal};
```

Comment:
The funky thing here is that whether or not something is considered a boolean attribute depends on both the element (tag) name and the attribute name. I figured we should precompute it for better minification/tree-shaking.